src/components/Greeting.js

```diff
@@ -0,0 +1,26 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { getGreeting } from './redux/greeting';
+import '../App.css';
+
+const Greeting = () => {
+  const message = useSelector((state) => state.greeting.greeting);
+  console.log(message);
```

Comment:
- [ ] While fixing the bug mentioned by the previous CR you put in a `console.log()` please avoid pushing this to your remote repo because it will make your codebase look unorganised and harder to read. please remove it before requesting another review.

Reply:
Bug Fixed and console.log removed